arbiter-core/src/environment/mod.rs

```diff
@@ -556,6 +562,18 @@ impl Environment {
                                     )),
                                 }
                             }
+                            EnvironmentData::TransactionCount(address) => {
+                                let db = evm.db().unwrap();
+                                let recast_address = revm::primitives::Address::from(address);
+                                match db.accounts.get(&recast_address) {
```

Comment:
```suggestion
                                match db.accounts.get(&address.into()) {
```

Reply:
So this gives some compile errors about needing additional type annotations. So it would need something like this
```rust
match db.accounts.get::<revm::primitives::Address>(&address.into())
```
Also I was following the convention from above that was used in the balance quarry. If we change it here and use the type annotations wouldn't it make sense to change everywhere?

Reply:
Sure, I was just suggesting something that is less LOC